app/helpers/hyrax/collections_helper.rb

```diff
@@ -24,6 +24,13 @@ def available_parent_collections_data(collection:)
       end.to_json
     end
 
+    ##
+    # @since 3.1.0
```

Comment:
This should be updated to 3.4.0?
```suggestion
    # @since 3.4.0
```